modules/distribution/product/src/main/conf/deployment.toml

```diff
@@ -92,12 +92,9 @@ https_endpoint = "https://localhost:${https.nio.port}"
 
 [apim.analytics]
 enable = false
-#reporter_impl="class.name"
-#handler_impl="class.name"
-#properties.'auth.api.url'="https://localhost:9443/auth"
-#properties.'token.api.url'="https://localhost:9443/token"
-#properties.'consumer.key'="consumer_key"
-#properties.'consumer.secret'="consumer_secret"
+config_endpoint = "https://localhost:8080/auth/v1"
+auth_token = ""
+properties.'worker.thread.count' = 1
```

Comment:
Why do we need to have it as a property?

Reply:
the event not publishing issue is due to the multi-threads. So as a temporary solution I'm limiting to single worker threads,